userdocs/src/usage/vpc-vpc-configuration.md

```diff
@@ -0,0 +1,224 @@
+with a fully-functional dedicated VPC.
+
+**Examples**
+
+Create a cluster using a custom VPC with 2x private and 2x public subnets:
+
+```
+eksctl create cluster \
+  --vpc-private-subnets=subnet-0ff156e0c4a6d300c,subnet-0426fb4a607393184 \
+  --vpc-public-subnets=subnet-0153e560b3129a696,subnet-009fa0199ec203c37
+```
+
+or use the following equivalent config file:
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: my-test
+  region: us-west-2
+
+vpc:
+  id: "vpc-11111"
+  subnets:
+    private:
+      us-west-2a:
+          id: "subnet-0ff156e0c4a6d300c"
+      us-west-2c:
+          id: "subnet-0426fb4a607393184"
+    public:
+      us-west-2a:
+          id: "subnet-0153e560b3129a696"
+      us-west-2c:
+          id: "subnet-009fa0199ec203c37"
+
+nodeGroups:
+  - name: ng-1
+```
+
+Create a cluster using a custom VPC with 3x private subnets and make initial nodegroup use those subnets:
+
+```
+eksctl create cluster \
+  --vpc-private-subnets=subnet-0ff156e0c4a6d300c,subnet-0549cdab573695c03,subnet-0426fb4a607393184 \
+  --node-private-networking
+```
+
+or use the following equivalent config file:
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: my-test
+  region: us-west-2
+
+vpc:
+  id: "vpc-11111"
+  subnets:
+    private:
+      us-west-2d:
+          id: "subnet-0ff156e0c4a6d300c"
+      us-west-2c:
+          id: "subnet-0549cdab573695c03"
+      us-west-2a:
+          id: "subnet-0426fb4a607393184"
+
+nodeGroups:
+  - name: ng-1
+    privateNetworking: true
+
+```
+
+Create a cluster using a custom VPC 4x public subnets:
+
+```
+eksctl create cluster \
+  --vpc-public-subnets=subnet-0153e560b3129a696,subnet-0cc9c5aebe75083fd,subnet-009fa0199ec203c37,subnet-018fa0176ba320e45
+```
+
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: my-test
+  region: us-west-2
+
+vpc:
+  id: "vpc-11111"
+  subnets:
+    public:
+      us-west-2d:
+          id: "subnet-0153e560b3129a696"
+      us-west-2c:
+          id: "subnet-0cc9c5aebe75083fd"
+      us-west-2a:
+          id: "subnet-009fa0199ec203c37"
+      us-west-2b:
+          id: "subnet-018fa0176ba320e45"
+
+nodeGroups:
+  - name: ng-1
+```
+
+More examples can be found in the repo's `examples` folder:
+
+- [using an existing VPC](https://github.com/weaveworks/eksctl/blob/master/examples/04-existing-vpc.yaml)
+- [using a custom VPC CIDR](https://github.com/weaveworks/eksctl/blob/master/examples/02-custom-vpc-cidr-no-nodes.yaml)
+
+## Custom Shared Node Security Group
+
+`eksctl` will create and manage a shared node security group that allows communication between
+unmanaged nodes and the cluster control plane and managed nodes.
+
+If you wish to provide your own custom security group instead, you may override the `sharedNodeSecurityGroup`
+field in the config file:
```

Comment:
```suggestion
If you wish to provide your own custom security group instead, you may override the `sharedNodeSecurityGroup` field in the config file:
```

Reply:
funny indentation 😅

Reply:
It's broken because otherwise it's one long sentence which some editors display in a single line which is hard to read. The rendered output doesn't care about the line break.

Reply:
ok as long as it looks good on the website I am ok :D

Reply:
vim people's line break. :D :D